coeus-it/src/test/java/org/kuali/kra/test/infrastructure/lifecycle/KcIntegrationTestBaseLifecycle.java

```diff
@@ -36,7 +36,7 @@
     
     public static final String BROWSER_PROTOCOL = "http";
     public static final String BROWSER_ADDRESS = "127.0.0.1";
-    public static final String PORTAL_ADDRESS = "kc-dev";
+    public static final String PORTAL_ADDRESS = "";
```

Comment:
When PORTAL_ADDRESS = "kc-dev", we can't access the image using the relative path. Its always need kc-dev at first of the path. Since the template is using across the application we can't change the path to kc-dev. So we removed.